src/Raven.Server/Documents/Expiration/ExpirationStorage.cs

```diff
@@ -175,15 +178,19 @@ public record ExpiredDocumentsOptions
                                         expiredDocs.Add((clonedId, id));
```

Comment:
dont we need to increase `totalCount` here as well?

Reply:
imo, regarding the main purpose of this feature (https://github.com/ravendb/ravendb/pull/18223#discussion_r1521284436) we should increment `totalCount` here too

Reply:
yeah, you are right! done